omc3/harpy/frequency.py

```diff
@@ -112,10 +112,21 @@ def harpy_per_plane(harpy_input, bpm_matrix, usv, tunes, plane):
     bpm_matrix = bpm_matrix.loc[df.index]
     spectra = {"FREQS": frequencies.loc[df.index], "COEFFS": coefficients.loc[df.index]}
 
-    if _get_natural_tunes(harpy_input, tunes) is not None:
-        df_nattunes = _calculate_natural_tunes(
-            spectra, _get_natural_tunes(harpy_input, tunes), harpy_input.tolerance, plane
-        )
+    nattunes = _get_natural_tunes(harpy_input, tunes)
+    tune_tol = harpy_input.tolerance
+    if nattunes is not None:
+        if any(abs(t - n) < tune_tol for n, t in zip(nattunes[:2], tunes[:2])):
+            if all(n == t for n, t in zip(nattunes[:2], tunes[:2])):
+                # Different error to try and be more informative
+                raise ValueError(
+                    "Natural tunes match driven tunes. "
+                    "If analysing a simulation without an ACD don't give the nattunes or natdeltas."
+                )
+            raise ValueError(
+                "Driven tunes exist within the natural tune tolerance."
+                "Please check the input parameters."
+            )
```

Comment:
Suggestion:

"At least one of the driven tunes is within the tolerance window of finding the natural tunes. Please check the input parameters."